native/c/zcnm.c

```diff
@@ -89,7 +97,13 @@ int ZCNGET(ZCN *zcn, char *response)
   }
 
   if (zcn->ecb)
+  {
+    int timeout = zcn->timeout * 100; // covert milliseconds to seconds
```

Comment:
If the intent is to convert ms to sec here, you'll want to use 1,000 as the constant:


```suggestion
    int timeout = zcn->timeout * 1000; // convert milliseconds to seconds
```

Reply:
ah ok, that's odd but fair enough haha 😋  thanks